src/client/screens/Feed/index.js

```diff
@@ -23,19 +23,26 @@ class FeedPage extends React.Component {
           'timeStamp': 1508469350963,
           'userPicURL': 'https://fm.cnbc.com/applications/cnbc.com/resources/img/editorial/2017/05/12/104466932-PE_Color.240x240.jpg?v=1494613853',
           'postPicURL': 'http://ronpaulinstitute.org/media/121032/donald-trumps-mexico-border-wall-557313.jpg',
-          'numOfLikes': 248,
-          'numOfComments': 36,
-          'numOfShares': 192,
+          'likes': [],
+          'comments': [{}],
+          'shares': [],
+          'numOfComments': 0,
+          'numOfLikes': 0,
+          'numOfShares': 0,
+
         }, {
           'postId': '2',
           'username': 'Donald Trump',
           'timeStamp': 1508469350963,
           'postText': 'Make America great again! #America #greatwall',
           'userPicURL': 'https://fm.cnbc.com/applications/cnbc.com/resources/img/editorial/2017/05/12/104466932-PE_Color.240x240.jpg?v=1494613853',
           'postPicURL': 'http://ronpaulinstitute.org/media/121032/donald-trumps-mexico-border-wall-557313.jpg',
-          'numOfLikes': 248,
-          'numOfComments': 36,
-          'numOfShares': 192,
+          'likes': [],
+          'comments': [{}],
```

Comment:
do not use invalid default value. It must be an empty array or a valid comment descriptor. In this case you can break your own application